src/test/java/edu/hm/hafner/util/StringContainsUtilsTest.java

```diff
@@ -30,5 +38,17 @@ void shouldSearchStrings() {
         assertThat(containsAnyIgnoreCase("This is a string text.", "this")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "is")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "IS")).isTrue();
+
+        assertThat(containsAllIgnoreCase("This is a string text.", "something")).isFalse();
+
+        assertThat(containsAllIgnoreCase("This is a string text.", "This")).isTrue();
+        assertThat(containsAllIgnoreCase("This is a string text.", "this")).isTrue();
+        assertThat(containsAllIgnoreCase("This is a string text.", "a", "is")).isTrue();
+        assertThat(containsAllIgnoreCase("This is a string text.", "A", "is")).isTrue();
+        assertThat(containsAllIgnoreCase("This is a string text.", "wrong", "is")).isFalse();
+        assertThat(containsAllIgnoreCase("This is a string text.", "wrong", "IS")).isFalse();
+        assertThat(containsAllIgnoreCase("This is a string text.", "wrong", null)).isFalse();
```

Comment:
Hier sinnvoller nur einen Fehler testen: 
```suggestion
        assertThat(containsAllIgnoreCase("This is a string text.", "This", null)).isFalse();
```